<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_AgregarEmpleado">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbx_ID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbx_Nombre">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbx_Departamento">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Departamento</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbx_Puesto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Puesto</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbx_Salario">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Salario(USD)</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_Output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>771</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Guardar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Cargar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Reporte">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar Reporte</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
